extension/src/popup/components/account/AssetDetail/index.tsx

```diff
@@ -296,30 +356,24 @@ export const AssetDetail = ({
               </div>
             </div>
           </div>
-          {sortedAssetOperations.length ? (
-            <div className="AssetDetail__list" data-testid="AssetDetail__list">
-              <>
-                {sortedAssetOperations.map((operation) => (
-                  <HistoryItem
-                    key={operation.id}
-                    accountBalances={accountBalances}
-                    operation={operation}
-                    publicKey={publicKey}
-                    networkDetails={networkDetails}
-                    setActiveHistoryDetailId={() =>
-                      setActiveAssetId(operation.id)
-                    }
-                  />
-                ))}
-              </>
-            </div>
-          ) : (
+          {assetOperations === null ? (
```

Comment:
if we are in the rare case where a user tries to view their Asset Details, but history has not finished loading yet, this allows us to just show a <Loader /> until that request finishes